homeassistant/components/rfxtrx.py

```diff
@@ -108,6 +109,15 @@ def _valid_light_switch(value):
     return _valid_device(value, "light_switch")
 
 
+def valid_recvmode(value):
+    """Test if a recv_mode value is known by pyRFXTRX."""
+    import RFXtrx
```

Comment:
@home-assistant/core : I think the RFXtrx library is installed after the config check? So this fail on a fresh install?

Reply:
I've tried and you are right. It does not work. I've moved the checks inside setup().